### PR TITLE
recommend shadow clone for end users

### PR DESCRIPTION
**Pull request by aminroosta**
cloning the repository takes a while.
for end users a shadow clone works just fine.

### CODE REVIEW

**Comment by zhengbli** [comment]
So this practically makes the end user clone the repo without the history? Interesting, and I agree for users that is normally enough. @steveluc @billti what do you think?

File: README.md [hunk 1]
```diff
@@ -15,14 +15,14 @@ look something like:
 
 ```
 cd ~/"Library/Application Support/Sublime Text 3/Packages"
-git clone https://github.com/Microsoft/TypeScript-Sublime-Plugin.git TypeScript
+git clone --depth=1 https://github.com/Microsoft/TypeScript-Sublime-Plugin.git TypeScript
```
Comment (zhengbli) [inline README.md]: Add a note to explain what ``--depth=1`` does. Something like this:
(``--depth=1`` downloads only the current version to reduce the clone size)

**Comment by mhegazy** [comment]
:+1:

**Comment by zhengbli** [comment]
Thanks for contributing!

**Pull request merged**